### ISSUE TITLE
Seatbelt Script

### ISSUE THREAD

**Issue by CharlieHunter1** [ScottUK/ladojrp-issues#106]
**Describe the feature you'd like implemented**
A seatbelt script like the one linked below (Got it in my Test Server it's sick) that shows a display when you have no seatbelt on, and makes a nice clicking noise when you put it on.

**Link to the resource if it already exists**
https://forum.cfx.re/t/release-seatbelt-updated/449900

**Additional context**
Would be cool, and  add realism :)

**Comment by KongeHvH** [comment]
I wouldn't see a point in this script unless if you could get ejected from your vehicle if you forgot your seat belt.

**Comment by CharlieHunter1** [comment]
@KongeHvH Yeah, the script does that. You get ejected upon crashes without a seatbelt.

**Comment by ladojrp-sourdirt** [comment]
Hello C. Hunter this is Sourdirt with the LADOJRP Suggestions team

Thank you for this suggestion and this will be read very carefully and taken under consideration.

I understand your idea of adding a different seatbelt script but it sounds like the only difference between the one currently in the server is the noise capability and adding back the option of people flying out the windshield of their car. I personally believe if that gets added back people will abuse it to the extent to crash into other's cars more often and would cause more people to break server rues. If you would like I can bump this up to the head of suggestion team and see what he thinks.

They will decide and make a vote on it and thank you again for reaching out to the suggestion team. These don't go unnoticed.

Administrator, Sourdirt

**Comment by CharlieHunter1** [comment]
Hello @ladojrp-sourdirt, the one currently in the server currently conflicts with the engine turn off script. It also doesn't launch players from the vehicle, or give a sound. I believe it will increase the RP  and need for people to drive properly if they forget to use their seatbelt because otherwise they'll be launched from the vehicle. I'd appreciate it being passed up, as I'd love to see this in the server!
